android/src/main/java/com/mastercard/compass/cp3/lib/react_native_wrapper/route/GetRegistrationDataAPIRoute.kt

```diff
@@ -0,0 +1,61 @@
+package com.mastercard.compass.cp3.lib.react_native_wrapper.route
+
+import android.app.Activity
+import android.content.Intent
+import android.util.Log
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReadableMap
+import com.mastercard.compass.cp3.lib.react_native_wrapper.R
+import com.mastercard.compass.cp3.lib.react_native_wrapper.ui.GetRegistrationDataCompassApiHandlerActivity
+import com.mastercard.compass.cp3.lib.react_native_wrapper.util.ErrorCode
+import com.mastercard.compass.cp3.lib.react_native_wrapper.util.Key
+import com.mastercard.compass.model.card.RegistrationStatusData
+
+
+class GetRegistrationDataAPIRoute(private val context: ReactApplicationContext, private val currentActivity: Activity?) {
+  companion object {
+    val REQUEST_CODE_RANGE = 700 until 800
+
+    const val GET_REGISTRATION_DATA_REQUEST_CODE = 700
+    private const val TAG = "GetRegistrationDataAPIRoute"
+  }
+
+  fun startGetRegistrationIntent(GetRegistrationDataParams: ReadableMap){
+    val reliantAppGUID: String = GetRegistrationDataParams.getString("reliantGUID")!!
```

Comment:
Please rename reliantAppGUID to reliantGUID for naming consistency across the files